clients/src/test/java/org/apache/kafka/common/network/SslSelectorTest.java

```diff
@@ -274,7 +274,7 @@ public void testRenegotiationFails() throws Exception {
             selector.poll(10);
             disconnected.addAll(selector.disconnected().keySet());
         }
-        assertTrue("Renegotiation should cause disconnection", disconnected.contains(node));
+        assertTrue(disconnected.contains(node), "Renegotiation should cause disconnection");
 
     }
 
```

Comment:
```testMuteOnOOM``` needs ```@Test```. Otherwise, it gets skipped.

Reply:
Good catch, seems like we haven't been running this test for a while. Luckily it still passes. :)